### PR TITLE
Updated docs for IdP login

### PR DESCRIPTION
**Pull request by condrenjustin**
added additional information regarding IdP initiated SAML login

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by apigirl — APPROVED** [review]

File: content/en/mobile/_index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ You can log in using standard authentication, Google authentication, or [SAML][3
 
 #### Enabling SAML
 
-SAML login requires you to set up and authenticate your SAML provider with Datadog. For SAML IdP-initiated login, refer to the note at the end of this section. To authenticate SAML:
+SAML login requires you to set up and authenticate your SAML provider with Datadog. To authenticate SAML:
```
Comment (daniel-m-dd) [inline content/en/mobile/_index.md:25]: The  flow for idp initiated login is still different then SP initiated SAML (entering the email to get the link wont give any valid link for IDP _only_ orgs)

Comment (condrenjustin) [reply content/en/mobile/_index.md:25]: What do you suggest I change here?

**Review by apigirl — APPROVED** [review]

File: content/en/mobile/_index.md (hunk 1 above)
Comment (daniel-m-dd) [reply content/en/mobile/_index.md:25]: Keeping it as before?Was there an issue with the original sentence？

Comment (condrenjustin) [reply content/en/mobile/_index.md:25]: I'll revert this change

**Pull request merged**